documentation/documentation/docs/evaluators/python_evaluator.md

```diff
@@ -0,0 +1,11 @@
+# Python evaluator
+## General usage
+This evaluator is responsible for running and executing tests on a student's Python code. juuw
+
+## Structure
+When submitting the project a teacher can add a requirments manifest `filename of manifest`, this way only the packages in the requirments file are installable on the evaluator.
+
+When no manifest is present students are able to install their own depedencies with a `requirments.txt` and a `dev-requirments.txt`.
```

Comment:
When no manifest is present, ...
requirments.txt -> requirements.txt

Reply:
same for dev

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078